kani-driver/src/metadata.rs

```diff
@@ -206,13 +195,24 @@ mod tests {
             mock_proof_harness("module::not_check_three", None, None),
         ];
         let ref_harnesses = harnesses.iter().collect::<Vec<_>>();
-        assert!(find_proof_harness("check_three", &ref_harnesses).is_err());
+        assert_eq!(
+            find_proof_harnesses(BTreeSet::from([&"check_three".to_string()]), &ref_harnesses)
```

Comment:
Curious: Are the harness names normalized before we try to find them?

Reply:
The harnesses `pretty_name` are fully qualified. Does that answer you question?

Reply:
Yes, thanks!